comCam/manual_defects/r22_s00/20241120T000000.ecsv

```diff
@@ -0,0 +1,33 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - name: x0
+#   unit: pix
+#   datatype: int32
+#   description: X coordinate of bottom left corner of box
+# - name: y0
+#   unit: pix
+#   datatype: int32
+#   description: Y coordinate of bottom left corner of box
+# - name: width
+#   unit: pix
+#   datatype: int32
+#   description: X extent of box
+# - name: height
+#   unit: pix
+#   datatype: int32
+#   description: Y extent of box
+# meta: !!omap
+# - OBSTYPE: defects
```

Comment:
This claims these are `defects`, but the path calls them `manual_defects`, and that is making the `writeCuratedCalibrations` code unhappy.
```
- OBSTYPE: manual_defects
- CALIBCLS: lsst.ip.isr.Defects
```
I think needs to be added to all of these files.

Reply:
Looking closer at the error, I think this also needs to have the instrument changed as well (I pulled this from one of the transmission curves):
```
# - {INSTRUME: ComCam}
```